test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -426,7 +426,12 @@ end
 
 function commonVehicleData.getMobileAppId(pAppId)
   if not pAppId then pAppId = 1 end
-  return config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  local regParams = config["application" .. pAppId].registerAppInterfaceParams
```

Comment:
@YarikMamykin I suppose SDL doesn't depends on version to handle full or not-full appId
There is an .INI parameter `UseFullAppID = true` which handle this

Reply:
@dboltovskyi , yes, SDL doesn't depend on version to handle full or not-full appID but the ATF does.
That is why this if-statement was added.

Reply:
@YarikMamykin Could you suggest where such dependency is implemented in ATF?

Reply:
@dboltovskyi , sorry for my misunderstanding of your question. 
I should make a remark - SDL cuts out _fullAppID_ in message that comes from mobile app if syncMsgVersion is lower than 5.0 accordingly to Mobile API:         
<param name="fullAppID" type="String" maxlength="100" mandatory="false" **since="5.0"**>

So to be correct - we have NO dependencies on fullAppID in ATF, but we have them in SDL accordingly to Mobile API.

Reply:
@YarikMamykin Thank you for explanation